### PR TITLE
Update DatasetError.vue

### PR DESCRIPTION
**Pull request by jennaj**
Update DatasetError troubleshooting FAQ URL: Hub > GTN

Addresses outdated link described here: https://github.com/galaxyproject/galaxy/issues/13144

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [x] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

File: client/src/components/DatasetInformation/DatasetError.vue [hunk 1]
```diff
@@ -43,7 +43,7 @@
                         <br />
                         Start here:
                         <b>
-                            <a href="https://galaxyproject.org/support/tool-error/" target="_blank">
+                            <a href="https://training.galaxyproject.org/training-material/faqs/galaxy/#troubleshooting-errors" target="_blank">
```
Comment (nsoranzo) [inline client/src/components/DatasetInformation/DatasetError.vue]: @hexylena Is this a better URL?

```suggestion
                            <a href="https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_troubleshooting.html" target="_blank">
```

Comment (jennaj) [reply client/src/components/DatasetInformation/DatasetError.vue]: I considered that -- but I wanted users to land in a place that had other faqs. The direct link doesn't give that context, and the faqs cannot be searched yet, so the direct page is a dead-end. Interested in what @hexylena thinks though

Comment (hexylena) [reply client/src/components/DatasetInformation/DatasetError.vue]: I could go either way, but it's a good point about very much being a 'dead end'. 

There are solutions to that, if we recommend more "related FAQs" on that page, or breadcrumbs to get back to the FAQ category, but, overall i think it's fine to just point to that page and revise later if the per-faq page becomes more exciting?

Comment (jennaj) [reply client/src/components/DatasetInformation/DatasetError.vue]: Agree with all of that, thanks! Let's use the original link with context for now, and think about how to add breadcrumbs. This is the first of several links within Galaxy to update. We can always change our mind later.

**Review by dannon — APPROVED** [review]

**Comment by jennaj** [comment]
Ok -- the direct link is improved now with context (thank you @hexylena && @shiltemann!!)

https://training.galaxyproject.org/training-material/faqs/galaxy/analysis_troubleshooting.html

The `22.01` release hasn't been announced yet. Should we change it? I can open another PR.

**Pull request merged**